overrides/ios-override.json

```diff
@@ -316,7 +316,24 @@
         },
         "sync": {
             "state": "enabled",
-            "minSupportedVersion": "7.104.0"
+            "minSupportedVersion": "7.104.0",
+            "features": {
+                "level0ShowSync": {
```

Comment:
These are also found in features/sync.json. If we just add the `seamlessAccountSwitching` flag, it'll override the others, disabling them. Therefore it's important to move these up to the override level.